lmp/bb-config.sh

```diff
@@ -10,7 +10,9 @@ AKLITE_TAG="${AKLITE_TAG-promoted}"
 H_BUILD="${H_BUILD-lmp-localdev}"
 LMP_VERSION="${LMP_VERSION-unknown}"
 FACTORY="${FACTORY-lmp}"
-LMP_DEVICE_API="${LMP_DEVICE_API-https://api.foundries.io/ota/devices/}"
+LMP_DEVICE_API="${LMP_DEVICE_API-https://api.$(dns_base)/ota/devices/}"
+LMP_OAUTH_API="${LMP_OAUTH_API-https://app.$(dns_base)/oauth}"
+FIO_HUB_URL="${FIO_HUB_URL-hub.$(dns_base)}"
```

Comment:
Maybe we should do the same for apps/publish?

Reply:
i think apps/publish handles this also?

Reply:
I just thought about adding those env. vars to `publish.sh`.